roles/env_op_images/tasks/main.yml

```diff
@@ -60,22 +60,22 @@
         oc get ClusterServiceVersion
         -l operators.coreos.com/openstack-operator.openstack-operators
         --all-namespaces
-        -o yaml |
-        yq e '
+        -o json |
+        jq -r '
         [.items[]? |
         .spec.install.spec.deployments[]? |
         .spec.template.spec.containers[]? |
         .env[]? |
         select(.name? | test("^RELATED_IMAGE")) |
-        select(.name == "*manager*" or .name == "*MANAGER*") |
-        {(.name): .value}]'
+        select(.name | contains("MANAGER")) |
```

Comment:
We don't care about lower and upper cases here anymore?

Otherwise I would expect here something like `|ascii_upcase` involved, or `test("manager"; "i")` instead... (tested on https://play.jqlang.org)

Reply:
so from the list all are uppercase

Reply:
ack, thanks for clarifying – I am all fine then with this :+1: